packages/logger/.npmignore

```diff
@@ -1 +0,0 @@
-/tests/
```

Comment:
@mh81  Presumably this is one of the packages we *do* still want to publish?

Reply:
There are a couple of ways to specify what files should or should not be included in your npm package. One way is to use a `.npmignore` file, which will add everything to the npm package except what is in `.npmignore`. Another option is to use the files property in your `package.json` file ([documented here](https://docs.npmjs.com/files/package.json#files)). Rather than adding everything by default and not including the stuff in `.npmignore`, using the `files` property will include nothing _except_ for what's listed in `files`. That seemed like a much more reliable way for us to only include the stuff that we care about.

To answer @Jkovarik's question, yes, this package is still being published.